python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
Tbf the most popular JS style (by far) is the AirBnB one and they advise for it, for the same reasons as Dan mentioned earlier https://github.com/airbnb/javascript#commas--dangling

I'd still like to keep things consistant between languages even if doing `x` in language `x` isn't as popular